src/Application/Commands/MarkOldPasswordedAccountsInStripe.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace MatchBot\Application\Commands;
+
+use Doctrine\DBAL\Connection;
+use Psr\Log\LoggerInterface;
+use Stripe\StripeClient;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MarkOldPasswordedAccountsInStripe extends LockingCommand
+{
+    public const IDENTITY_DBAL_CONNECTION_SERVICE_NAME = 'identity_dbal_connection';
+
+    protected static $defaultName = 'matchbot:mark-old-passworded-accounts-in-stripe';
+    private LoggerInterface $logger;
+
+    public function __construct(
+        LoggerInterface $logger,
+        private readonly StripeClient $stripeClient,
+        private \Redis $redis,
+        private Connection $identityDBConnection,
+    ) {
+        $this->logger = $logger;
+        $this->setLogger($logger);
+        parent::__construct();
+    }
+
+    protected function doExecute(InputInterface $input, OutputInterface $output): int
+    {
+        $personRows = $this->identityDBConnection->fetchAllAssociative(
+            'SELECT BIN_TO_UUID(id) as uuid, updated_at, stripe_customer_id from Person where Password IS NOT NULL AND created_at < "2023-03-01" LIMIT 1'
```

Comment:
As discussed, should make sure the date range is inclusive (i.e. do the same second an additional time to be safe) and the order is explicit, when we use Redis-stored datetimes